tests/jsx_template/styled-components.js

```diff
@@ -0,0 +1,11 @@
+<style jsx>{`
+  p {
+    color: red;
+  }
+`}</style>;
+
+<style jsx>{tpl`
+  p {
+    color: red;
+  }
+`}</style>;
```

Comment:
Good call. I explicitely didn't cover this case. If there are any spaces, then I just do whatever it used to do.